### PR TITLE
React to removal of [Activate] for ViewComponents

### PR DESCRIPTION
**Pull request by rynowak**

### CODE REVIEW

File: .vs/config/applicationhost.config [hunk 1]
```diff
@@ -0,0 +1,1036 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    IIS configuration sections.
```
Comment (rynowak) [inline .vs/config/applicationhost.config]: Will undo this.

Comment (dougbu) [inline .vs/config/applicationhost.config]: probably worth updating the .gitignore file in this repo

**Comment by dougbu** [comment]
:shipit:

**Comment by Eilon** [comment]
The code seems soooooo much cleaner now! (Seriously!)

**Comment by kichalla** [comment]
c428ddffa3aebebf42b43c4718ec9ff2b398a469

**Pull request closed without merging**